app/server/routes/v1/comment/commentVoting.js

```diff
@@ -0,0 +1,138 @@
+/* 
+ Copyright © 2022 Province of British Columbia
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+/**
+ * Application entry point
+ * @author [Brady Mitchell](braden.jr.mitch@gmail.com)
+ * @module
+ */
+
+const express = require("express");
+
+const router = express.Router();
+
+const Comment = require("../../../models/comment.model");
+const User = require("../../../models/user.model");
+
+/**
+ * @swagger
+ * paths:
+ *  /api/comment/vote/{id}:
+ *    patch:
+ *      security:
+ *        - bearerAuth: []
+ *      tags:
+ *        - Comment Voting
+ *      summary: Upvote/downvote or unset vote comment.
+ *      description: Use query parameter "vote" with values "up", "down", or "unset".
+ *      parameters:
+ *        - in: path
+ *          name: id
+ *          required: true
+ *          schema:
+ *            $ref: '#/components/schemas/Comment/properties/id'
+ *        - in: query
+ *          name: vote
+ *          required: true
+ *          example: up
+ *      responses:
+ *        '404':
+ *          description: User not found. **||** <br>Comment not found. **||** <br>Vote not found in query.
+ *        '403':
+ *          description: Missing message in body of the request. **||** <br>Must be a part of community to post in community.
+ *        '204':
+ *          description: Success.
+ *        '400':
+ *          description: Bad Request.
+ */
+
+// Upvote/downvote or unset vote comment
+router.patch("/:id", async (req, res) => {
```

Comment:
I have to say I really like how readable this, and pretty much everything else you've done, is. Great work imo!

Reply:
Thanks! It could be better when I look back on some of it, but in the moment I try to make it as readable as I can.